packages/dds/sequence/src/revertibles.ts

```diff
@@ -387,11 +387,22 @@ function getSlidePosition(
 	const slide = collection.getSlideToSegment(lref);
 	return slide?.segment !== undefined &&
 		slide.offset !== undefined &&
-		string.getPosition(slide.segment) !== -1
+		string.getPosition(slide.segment) !== -1 &&
+		(pos < 0 || pos > string.getLength())
```

Comment:
pos = string.getLength() would be off the end and then isValidRange would fail. (Shouldn't affect fuzz test because it keeps the last char.)

```suggestion
		(pos < 0 || pos >= string.getLength())
```

Reply:
Since the remove is exclusive on the endpoint, wouldn't string.getLength() be a valid end position? Or is this change just to prevent the last char from being included in the removed range?